### PR TITLE
Basic rules and printing

### PR DESCRIPTION
**Pull request by charypar**
Add the basic grid system, rules of the game and initial command line printing.

### CODE REVIEW

File: src/game_of_life/game.clj [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      (map
+        (fn [y]
+          (vec
+            (map
+              (fn [x] (map-fn x y (get-cell grid x y)))
+              (range w))))
+        (range h)))))
+
+(defn random-grid
+  "Creates a random grid with a probability of cell"
```
Comment (samwhite) [inline src/game_of_life/game.clj]: can probably word this better 😄

File: src/game_of_life/game.clj [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        (cond
+          (and (true? v) (< n 2)) false ; cell with less than two neighbours dies
+          (and (true? v) (> n 3)) false ; cell with four or more neighbours dies
+          (and (false? v) (= n 3)) true ; three neighbours create a cell
+          :else v))) ; else cell stays the same
+    grid))
+
+(defn to-string
+  "Print a grid as a multi-line string"
+  [grid cell empty]
```
Comment (samwhite) [inline src/game_of_life/game.clj]: can probably rename `cell` in this context to "alive"

**Review by samwhite — COMMENTED** [review]
👍

**Review by romanschejbal — APPROVED** [review]
I approve diz goodness! 🎉  👍

**Comment by charypar** [comment]
@samwhite Changed the naming and wording per your comments.

**Review by samwhite — APPROVED** [review]
LGTM 👍

**Pull request merged**